model/flow/constants.go

```diff
@@ -28,6 +28,10 @@ const DefaultTransactionExpiryBuffer = 30
 // DefaultMaxTransactionGasLimit is the default maximum value for the transaction gas limit.
 const DefaultMaxTransactionGasLimit = 9999
 
+// EstimatedComputationPerMillisecond is the approximate number of computation units that can be performed in a millisecond.
+// this was calibrated during the Variable Transaction Fees: Execution Effort FLIP https://github.com/onflow/flow/pull/753
+const EstimatedComputationPerMillisecond = 9999.0 / 200.0
```

Comment:
Is this estimated as `MaxTransactionGasLimit / AverageDurationForTransactionWithMaxGas`?

If the accuracy is not that important, why not `10000 / 200` ?

Reply:
> Is this estimated as MaxTransactionGasLimit / AverageDurationForTransactionWithMaxGas?

yes. The AverageDurationForTransactionWithMaxGas is an old measurement (the one done during the calibration of the execution weights)

> If the accuracy is not that important, why not 10000 / 200 ?

Its not that important, but I do't know if we benefit from rounding here.